internal/experiment/generation/stormforger.go

```diff
@@ -112,28 +139,25 @@ func (s *StormForgerSource) Read() ([]*yaml.RNode, error) {
 
 		cm := &corev1.ConfigMap{}
 		cm.Name = s.stormForgerConfigMapName()
-		// TODO filepath.Base is broken here if TestCaseFile is a URL with a query parameter
-		cm.Data = map[string]string{filepath.Base(s.Scenario.StormForger.TestCaseFile): string(data)}
+		cm.Data = map[string]string{tc + ".js": string(data)}
 		result = append(result, cm)
 	}
 
 	// Include a secret with the access token, if necessary
-	if accessToken := s.stormForgerAccessToken(); accessToken != nil {
+	if accessToken := s.stormForgerAccessToken(org); accessToken != nil {
```

Comment:
With including the secretRef above in the pod spec, don't we always want to run through this flow regardless?
Im concerned we may silently allow a broken configuration that won't be immediately obvious ( if a trial pod doesn't have the secret available, is that going to just hang indefinitely ?)

Reply:
If you put an explicit secret ref in the app.yaml, the assumption is that we can't verify it's existence. For example, you could have this in your app.yaml:
```
stormforger:
  accessToken:
    secretKeyRef:
      name: so-much-secret
      key: no-peeking
```
And we would generate the trial pod blindly referencing those values. This actually closes the gap were no credential information is available: previously that would have still produce a reference even though there was no expectation of the secret being generated or instructions from the user that it pre-exists.